rules/predicates.py

```diff
@@ -211,9 +216,12 @@ def _apply(self, *args):
             callargs = (self,) + callargs
         try:
             result = self.fn(*callargs)
-            return None if result is None else bool(result)
+            returner = None if result is None else bool(result)
         except SkipPredicate:
-            return None
+            returner = None
+        
+        logger.debug('%s = %s', self, 'skipped' if returner is None else returner)
```

Comment:
I think it would be nice to indent by two spaces this one, so that output is visually parsed as a group:

```
Testing ((requires_two_args & requires_two_args) & passthrough)
  requires_two_args = skipped
  requires_two_args = skipped
  (requires_two_args & requires_two_args) = skipped
  passthrough = True
  ((requires_two_args & requires_two_args) & passthrough) = True
```

I would also change the equals sign to `=>` or anything you feel that can convey direction, i.e. "this resulted in that".